nuxeo-jsf/nuxeo-platform-ui-web/src/main/java/org/nuxeo/ecm/platform/ui/web/tag/fn/Functions.java

```diff
@@ -196,7 +196,13 @@ public static String joinArrayWithFinalDelimiter(Object[] collection, String sep
     public static String formatDateUsingBasicFormatter(Date date) {
         return formatDate(date, basicDateFormatter());
     }
-
+    
```

Comment:
Here you added trailing spaces. Please configure your IDE to remove all trailing spaces on save.